holidays/countries/cocos_islands.py

```
@@ -0,0 +1,237 @@
+
+        # Eid al-Fitr.
+        for dt in self._add_eid_al_fitr_day(tr("Eid al-Fitr")):
+            self._add_observed(dt, tr("Eid al-Fitr"))
+
+        # Eid al-Adha.
+        for dt in self._add_eid_al_adha_day(tr("Eid al-Adha")):
+            self._add_observed(dt, tr("Eid al-Adha"))
+
+
+class CocosIslandsIslamicHolidays(_CustomIslamicHolidays):
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)